src/server/game/Entities/Player/PlayerUpdates.cpp

```diff
@@ -0,0 +1,2058 @@
+#include "BattlefieldMgr.h"
+#include "Channel.h"
+#include "ChannelMgr.h"
+#include "Formulas.h"
+#include "GridNotifiers.h"
+#include "Group.h"
+#include "Guild.h"
+#include "InstanceScript.h"
+#include "Language.h"
+#include "OutdoorPvPMgr.h"
+#include "Pet.h"
+#include "Player.h"
+#include "SavingSystem.h"
+#include "ScriptMgr.h"
+#include "SkillDiscovery.h"
+#include "SpellAuraEffects.h"
+#include "UpdateFieldFlags.h"
+#include "Vehicle.h"
+#include "WeatherMgr.h"
+#include "CellImpl.h"
+#include "GridNotifiers.h"
+#include "GridNotifiersImpl.h"
+
+// Zone Interval should be 1 second
+#define ZONE_UPDATE_INTERVAL (1 * IN_MILLISECONDS)
+
+#define CINEMATIC_UPDATEDIFF 500
+#define CINEMATIC_LOOKAHEAD (2 * IN_MILLISECONDS)
+
+void Player::Update(uint32 p_time) {
```

Comment:
please keep the same codestyle (brackets one the new line)

Reply:
i put them all on a new line for all functions but not else if, or switch statements since that would be tedious to do manually, or if you want me to i can run the file over with clang-format with the pr i made and see how it turns out.

Reply:
@iraizo there is no need to change the codestyle at this moment, just cut and paste the code as it is from `Player.cpp`

I think your editor probably has changed it automatically